src/brush.rs

```diff
@@ -199,6 +203,16 @@ impl Brush {
                 LineDirection::Free => self.curr,
                 LineDirection::Horizontal => Point2::new(self.curr.x, start.y),
                 LineDirection::Vertical => Point2::new(start.x, self.curr.y),
+                LineDirection::AngleSnap(snap) => {
```

Comment:
This does kind of make the Horizontal and Vertical modes redundant

Reply:
Indeed. How about just having `:brush/set line` default to `0` (no snapping), and if you want horizontal or vertical you just use `90`? So then we get rid of `LineDirection` and just have an angle or `Option<u32>`?

Reply:
Makes sense. Removed.